src/main/java/Deadlines.java

```diff
@@ -0,0 +1,27 @@
+public class Deadlines extends Task{
+
+    protected String by;
+
+    public Deadlines(String description, String by) {
+
+        super(description);
+        this.by = by;
+    }
+
+    public String getByDate(){
```

Comment:
maintain consistent spacing between ) and { (ideally it should have a space in between)

same applies for classes